client/src/app/tabs/cloud-bpmn/BpmnEditor.js

```diff
@@ -219,15 +219,18 @@ export class BpmnEditor extends CachedComponent {
       'propertiesPanel.focusout',
       'directEditing.activate',
       'directEditing.deactivate',
-      'searchPad.closed',
-      'searchPad.opened',
-      'popupMenu.opened',
-      'popupMenu.closed',
-      'feelPopup.opened',
-      'feelPopup.closed',
-      'feelPopup.focusin',
-      'feelPopup.focusout',
-      'elementTemplates.select'
+
+      // 'searchPad.closed',
+      // 'searchPad.opened',
+      // 'popupMenu.opened',
+      // 'popupMenu.closed',
+      // 'feelPopup.opened',
+      // 'feelPopup.closed',
+      // 'feelPopup.focusin',
+      // 'feelPopup.focusout',
+      'elementTemplates.select',
```

Comment:
```suggestion
```

Not needed, as this is also a "Focus changed, please update menu" event, like all other, cf. [implementation](https://github.com/search?q=org%3Abpmn-io%20elementTemplates.select&type=code)).